### ISSUE TITLE
New release expectation?

### ISSUE THREAD

**Issue by tyctor** [keleshev/schema#308]
Hi do you plan new release? Or how to install your package from github repo? 
when i put `schema@git+https://github.com/keleshev/schema.git` to my `pyproject.toml` 
or i do 
```
pip install schema@git+https://github.com/keleshev/schema.git
```
it is not being installed correctly (only `schema-0.7.5.-dist-info` folder, but no `schema.py` file)

**Comment by evgenii-moriakhin** [comment]
same problem with install from direct reference

I think this is due to the presence of pyproject.toml file with no metadata about the package installation

**Comment by skorokithakis** [comment]
Yep, I'll see if I can make a new release!

**Comment by skorokithakis** [comment]
Released 0.7.6, please let me know if there are any issues.

**Comment by skorokithakis** [comment]
I yanked the release as it breaks the import. If someone can fix this error, I can release again.
